integrations/ashby/actions/interview-stage.ts

```diff
@@ -0,0 +1,25 @@
+import type { InterviewStageList, InterviewStageListResponse, NangoAction, StagesResponse } from '../../models.js';
+import type { PaginationParams } from '../helpers/pagination.js';
+import paginate from '../helpers/pagination.js';
+
+export default async function runAction(nango: NangoAction, input: InterviewStageList): Promise<StagesResponse> {
+    return saveAllStages(nango, input.interviewPlanId);
```

Comment:
Would simplify this template. No need for the incorrectly named `saveAllStages` method. Just put the implementation in the `runAction` function